imagebuilder/main.go

```diff
@@ -94,13 +94,19 @@ func main() {
 		glog.Exitf("Error loading config: %v", err)
 	}
 
+	for key, value := range splitAdditionalTags() {
```

Comment:
The other option is to support a repeated flag (rather than parsing it) but that's a little awkward with the standard flag library (e.g. https://stackoverflow.com/questions/28322997/how-to-get-a-list-of-values-into-a-flag-in-golang ) so this works :-)